files/KoreBuild/modules/sharedsources/sharedsources.csproj

```diff
@@ -42,11 +42,11 @@
   <ItemGroup Condition="'$(NuspecBasePath)'!=''">
     <Compile Include="$(NuspecBasePath)**\*.cs" Exclude="$(DefaultExcludeItems)">
       <Pack>true</Pack>
-      <PackagePath>$(ContentTargetFolders)\cs\netstandard1.0\</PackagePath>
+      <PackagePath>$(ContentTargetFolders)\cs\netstandard1.0\$(PackageId)\%(RecursiveDir)\</PackagePath>
     </Compile>
     <EmbeddedResource Include="$(NuspecBasePath)**\*.resx" Exclude="$(DefaultExcludeItems)">
       <Pack>true</Pack>
-      <PackagePath>$(ContentTargetFolders)\any\any\</PackagePath>
+      <PackagePath>$(ContentTargetFolders)\any\any\$(PackageId)\%(RecursiveDir)\</PackagePath>
```

Comment:
What's this about? I don't see where PackagePath is used. Is this now basically PackageContentPath*s*?

Reply:
This is just some Solution Explorer sugar. It will make the make share source files appear in a folder that matches the .Sources package ID.

Reply:
That's pretty cool.